recipes-qt/qt5/qt5-git.inc

```diff
@@ -2,7 +2,7 @@
 # Copyright (C) 2013-2020 Martin Jansa <martin.jansa@gmail.com>
 
 QT_MODULE ?= "${BPN}"
-QT_MODULE_BRANCH ?= "5.15"
+QT_MODULE_BRANCH ?= "5.15.2"
```

Comment:
Aren't these 5.15.x branches short lived? IIRC such branches were sometimes removed later and only x.y branches were meant to be permanent (but not all SRCREVs now used were on 5.15 branches last time I've checked).

see
https://github.com/meta-qt5/meta-qt5/pull/69
https://github.com/meta-qt5/meta-qt5/pull/256
https://github.com/meta-qt5/meta-qt5/pull/110

Reply:
Qt has moved to cherry-pick mode instead of doing merges. The release branches (5.15.x) are never removed and the release sha1s are available only from the release branches.

Reply:
```
qtbase $ git remote prune origin
Pruning origin
URL: git://code.qt.io/qt/qtbase.git
 * [pruned] origin/5.11.0
 * [pruned] origin/5.11.1
 * [pruned] origin/5.11.2
 * [pruned] origin/5.11.3
 * [pruned] origin/5.12.0
 * [pruned] origin/5.12.1
 * [pruned] origin/5.12.10
 * [pruned] origin/5.12.11
 * [pruned] origin/5.12.12
 * [pruned] origin/5.12.2
 * [pruned] origin/5.12.3
 * [pruned] origin/5.12.4
 * [pruned] origin/5.12.6
 * [pruned] origin/5.12.8
 * [pruned] origin/5.12.9
 * [pruned] origin/5.13.0
 * [pruned] origin/5.13.1
 * [pruned] origin/5.13.2
 * [pruned] origin/5.14.0
 * [pruned] origin/5.14.1
 * [pruned] origin/5.14.2
 * [pruned] origin/5.15.0
 * [pruned] origin/5.15.1
 * [pruned] origin/5.15.2
 * [pruned] origin/5.9.6
 * [pruned] origin/5.9.7
 * [pruned] origin/5.9.8
 * [pruned] origin/6.0.0
 * [pruned] origin/6.1.0
 * [pruned] origin/6.1.1
 * [pruned] origin/6.1.2
 * [pruned] origin/6.1.3
 * [pruned] origin/6.2.1
 * [pruned] origin/6.2.2
 * [pruned] origin/6.2.3
 * [pruned] origin/6.3.0
 * [pruned] origin/6.3.1
 * [pruned] origin/6.3.2
 * [pruned] origin/compiledb
 * [pruned] origin/winrt
 * [pruned] origin/wip/protobuf
 * [pruned] origin/wip/qbs
 * [pruned] origin/wip/qbs2
 * [pruned] origin/wip/qt6
 * [pruned] origin/wip/qt6_snapshot
 * [pruned] origin/wip/yaml-dependencies
```

Reply:
> > Qt has moved to cherry-pick mode instead of doing merges. The release branches (5.15.x) are never removed and the release sha1s are available only from the release branches.
> 
> "never removed" until that changed with: https://codereview.qt-project.org/c/qt/qtqa/+/444499 and now many meta-qt5 branches and meta-qt6 branches are broken and I'm not going to add global nobranch=1 like meta-qt6 is adding in: https://codereview.qt-project.org/c/yocto/meta-qt6/+/447775

never say never; this lasted two years and they are gone again: [QTQAINFRA-3091](https://bugreports.qt.io/browse/QTQAINFRA-3091)
Merges are no longer done, so the release sha1s are not available from any branch at the moment, only through to the tags.

I know I've asked this before, but what makes you so adamantly against nobranch=1?

Reply:
Use the tag.

Reply:
@thiagomacieira SRCREVs are using git hashes of tags, that's not the issue, see https://github.com/meta-qt5/meta-qt5/issues/495#issuecomment-1398195630